tutorials/users/catalog/gui-search.md

```diff
@@ -1 +1,115 @@
-# gui-search
+# Catalog access via STAC Browser
+
+Each node in the Marble network include a GUI application called [STAC Browser](https://github.com/radiantearth/stac-browser) that allows you to 
+explore the STAC catalog of data that is available on that node. This application is available at `https://host-url/stac-browser`, where
+`host-url` is the URL of the Marble node. For example, for the Marble node 'Red Oak' available at 'https://daccs.cs.toronto.edu/',
+the STAC Browser can be accessed at <https://daccs.cs.toronto.edu/stac-browser>.
+
+In this tutorial, you will get an introduction to the STAC Browser, how to navigate within the Browser, how to perform search,
+and how to retrieve links that allow you to the data via one or more protocols. At the end of the section, we discuss some limitations with the GUI, 
+and programmatic way to interact with the STAC Catalog which is introduced in the [next section](pystac-client.ipynb).
+
+(gui_overview)=
+## GUI overview
+
+In this tutorial, we'll use Red Oak node as an example. The concepts introduced here apply to all STAC Browsers. 
+Start by navigating to the [STAC Browser on Red Oak](https://daccs.cs.toronto.edu/stac-browser) in a new tab; 
```

Comment:
Another reason is that this tutorial is meant for all users, not just those with an account on Red Oak. 

If a user wants to use this tutorial and follow along on another node they need to know what the appropriate URL is for their node. Here, the user doesn't actually see the URL, they just see that there is a clickable link (which takes them to a page they don't have access to).

The user knows that we're using Red Oak as an example but it's not immediately obvious that they need to modify the link for their use case because they can't see the link. We can change this to something like:

> Start by navigating to the STAC Browser on Red Oak at <https://daccs.cs.toronto.edu/stac-browser> in a new tab;

That way the structure of the URL is visible to the user.

Same below we can change bullet point 3 to:

> In this case it is displaying the _description_ field of the root catalog (at https://daccs.cs.toronto.edu/stac).

Reply:
Done